guides/source/install_ruby_on_rails.md

```diff
@@ -0,0 +1,133 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
+
+Install Ruby on Rails Guide
+=====================
+
+This guide will walk you through installing the Ruby programming language and the Rails framework on your operating system.
+
+While your OS might come with Ruby pre-installed, it's often outdated and can't be upgraded. Using a version manager like [Mise](https://mise.jdx.dev/getting-started.html) allows you to install the latest Ruby version, use a different Ruby version for each app, and easily upgrade to new versions when they're released.
```

Comment:
I've never heard of mise until today, why not use something most Ruby developers use or are familiar with like `rbenv`?

Reply:
This is what's used in Omakub and what DHH approved to use here.